modules/contracts/src.ts/services/ethService.ts

```diff
@@ -244,24 +241,22 @@ export class EthereumChainService extends EthereumChainReader implements IVector
   /// Throws ChainError if signer not found, tx not found, or tx already mined.
   public async getTxResponseFromHash(
     chainId: number,
-    tx: MinimalTransaction & { transactionHash: string; nonce: number },
+    txHash: string,
   ): Promise<Result<{ response?: TransactionResponse; receipt?: TransactionReceipt }, ChainError>> {
     const provider = this.chainProviders[chainId];
     if (!provider) {
       return Result.fail(new ChainError(ChainError.reasons.ProviderNotFound, { chainId }));
     }
 
     try {
-      const response = await provider.getTransaction(tx.transactionHash);
+      const response = await provider.getTransaction(txHash);
       let receipt: TransactionReceipt | undefined;
       if (response?.confirmations > 0) {
-        receipt = await provider.send("eth_getTransactionReceipt", [tx.transactionHash]);
+        receipt = await provider.send("eth_getTransactionReceipt", [txHash]);
       }
       return Result.ok({ response, receipt });
     } catch (e) {
-      return Result.fail(
-        new ChainError(ChainError.reasons.TxNotFound, { error: e, transactionHash: tx.transactionHash }),
-      );
+      return Result.fail(new ChainError(ChainError.reasons.TxNotFound, { error: e, transactionHash: txHash }));
     }
   }
 
```

Comment:
If we are removing speed up tx RPC method, lets remove this one as well? @rhlsthrm